pkg/knuu/instance_helper.go

```diff
@@ -531,3 +570,43 @@ func (i *Instance) addOtelCollectorSidecar() error {
 	}
 	return nil
 }
+
+func (i *Instance) createNetworkConfigInstance() (*Instance, error) {
+	networkConfigInstance, err := NewInstance("network-config")
+	if err != nil {
+		return nil, fmt.Errorf("error creating network-config instance: %w", err)
+	}
+	if err := networkConfigInstance.SetImage("ttl.sh/41838c59-c02e-4cac-9284-e3f47d32391d:latest"); err != nil {
+		return nil, fmt.Errorf("error setting image for network-config instance: %w", err)
+	}
+	if err := networkConfigInstance.Commit(); err != nil {
+		return nil, fmt.Errorf("error committing network-config instance: %w", err)
+	}
+	return networkConfigInstance, nil
+}
+
+func (i *Instance) addNetworkConfigSidecar() error {
+	networkConfigSidecar, err := i.createNetworkConfigInstance()
+	if err != nil {
+		return fmt.Errorf("error creating network config instance '%s': %w", i.k8sName, err)
+	}
+	bittwisterCmd := []string{"./bittwister", "start", "-d", "eth0"}
```

Comment:
This should not be hard coded. 
bittwister will be running as a service and knuu communicates through API calls via an SDK.

Reply:
This is temporary until the SDK with the API is ready.

Reply:
perfekt, let's give it a try and see how it goes while I am working on the API thing